dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostFactoryImpl.java

```diff
@@ -649,6 +650,21 @@ public Optional<List<Host>> findStoppedSites(final String siteNameFilter, final
         return search(siteNameFilter, condition, showSystemHost, limit, offset, user, respectFrontendRoles);
     }
 
+    @Override
+    public Optional<List<Host>> findLiveAndStopped(final String siteNameFilter,
```

Comment:
any test for this method?

Reply:
Tests methods added as requested
https://github.com/dotCMS/core/pull/25166/files#diff-0a135b189d8372722e5f6b598e21230edad8d45fb3c300ce4ef1569da0993cf5